shared/constants/favorite.js

```diff
@@ -6,4 +6,7 @@ import type {Props as FolderProps} from '../folders/render'
 export const favoriteList = 'favorite:favoriteList'
 export type FavoriteList = TypedAction<'favorite:favoriteList', {folders: FolderProps}, void>
 
-export type FavoriteAction = FavoriteList
+export const favoriteIgnore = 'favorite:favoriteIgnore'
+export type FavoriteIgnore = TypedAction<'favorite:favoriteIgnore', {error: ?string}, void>
```

Comment:
Why not use `error: true` ?

https://github.com/keybase/client/blob/master/shared/constants/types/flux.js#L3